signature/signature.go

```diff
@@ -16,6 +16,9 @@ import (
 
 const (
 	signatureType = "atomic container signature"
+
+	// Path pointing to a passphrase file when signing via GPGME.  Specified as an evironment variable.
```

Comment:
GPGME should not be a part of the API. “Name of an environment variable that contains …”.